visualizer/stack-bar.css

```diff
@@ -2,6 +2,7 @@
   position: relative;
   cursor: pointer;
   box-shadow: 0px -1px 0px rgba(255, 255, 255, 0.3) inset;
+  overflow: hidden;
```

Comment:
What's this for? Maybe I've missed something but I can't see what it adds

Reply:
@AlanSl This prevents vertical scrollbar from appearing when the user selects the last frame on the stack-bar. If the last frame is very close to the end of the stack-bar the pointer can overflow on the right triggering scrollbars.
Also resizing the window can trigger the same behaviour as the `reDrawStackBar` is debounced and can be larger than the viewport width for a fraction of time